src/components/cqlLibraryList/CqlLibraryList.tsx

```diff
@@ -142,6 +142,7 @@ export default function CqlLibraryList({ cqlLibraryList, onListUpdate }) {
   const [anchorEl, setAnchorEl] = useState(null);
   const [selectedCQLLibrary, setSelectedCqlLibrary] =
     useState<CqlLibrary>(null);
+  const isOwner = checkUserCanEdit(selectedCQLLibrary?.createdBy, []);
```

Comment:
Just a nitpick: I feel like isOwner might not be an appropriate name anymore, but that's something we can adjust later